sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -23,6 +23,19 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+VALID_CREDENTIALS = [
+    "DEVELOPER_CLI",
+    "WORKLOAD_IDENTITY",
+    "CLI",
+    "ENVIRONMENT",
+    "MANAGED_IDENTITY",
+    "POWERSHELL",
+    "VISUAL_STUDIO_CODE",
```

Comment:
I recommend excluding this credential until it works again

Reply:
To make sure we are on the same page, this means if user puts "VISUAL_STUDIO_CODE" in the environment, we will raise ValueError.

Reply:
Correct. We don't want to drive adoption of something that's broken.